### PR TITLE
Add support for Pmod-TC1

### PR DESCRIPTION
**Pull request by tfors**
Added support for Pmod-TC1 by starting from Pmod-ALS as example.

### CODE REVIEW

**Comment by schelleg** [comment]
Thanks for submitting driver - give me a couple of days to verify (I have to buy the device!).

**Comment by tfors** [comment]
No problem.

File: Pynq-Z1/sdk/pmod_tc1/src/pmod_tc1.c [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+#define LOG_CAPACITY  (4000/LOG_ITEM_SIZE)
+
+u32 get_sample(){
+  /*
+   * TC1 data is packed in a 32-bit word.
+   * Four bytes need to be read.
+   */
+  u8 raw_data[4];
+  spi_transfer(SPI_BASEADDR, 4, raw_data, NULL);
+  u32 v = ( (raw_data[0] << 24) + (raw_data[1] << 16) + (raw_data[2] << 8) + raw_data[3] );
```
Comment (yunqu) [inline Pynq-Z1/sdk/pmod_tc1/src/pmod_tc1.c]: Hi @tfors 

Line 75 seems to exceed the 79 characters. Can you please adjust it? 

We will probably upload some guidelines later.

Thanks,

Rock

Comment (tfors) [inline Pynq-Z1/sdk/pmod_tc1/src/pmod_tc1.c]: This should be fixed now.  Thanks for pointing it out!

**Comment by tfors** [comment]
Hi @beja65536,

Oops.  I modified the init file on my pynq to make it work and forgot to mod the one in the repo.  I've fixed that now and I've also added a legend and some axis labels to the plot in the example notebook.

Thanks for reviewing this.
Tom

**Pull request merged**